tubular/nominal.py

```diff
@@ -747,6 +747,12 @@ def fit(self, X, y):
 
         return self
 
+    def map_imputation_values(self, X):
+        for c in self.columns:
+            X[c] = X[c].map(self.mappings[c])
```

Comment:
I know it's a bit tangential but please could you rewrite the docstring for the self.mappings attribute, it's quite difficult to understand.

Reply:
https://github.com/lvgig/tubular/pull/135/commits/48aa1075cd59809aa0b918791a82d78078626182